### PR TITLE
Opt Out copy changes, fixes #11221

### PR DESCRIPTION
**Pull request by erikphansen**
Please let me know the preferred way to link a PR to a Zenhub issue. Just putting the issue number in the PR headline works but it doesn't look like that's what other folks are doing.

Screenshots of the changes:

screen shot 2018-07-23 at 4 02 38 pm
screen shot 2018-07-23 at 4 04 48 pm

### CODE REVIEW

File: src/platform/forms/sass/_m-schemaform.scss
Comment (erikphansen) [inline src/platform/forms/sass/_m-schemaform.scss]: This prevents the phone number from wrapping across two lines in the modal. This behavior is approved by @emilywaggoner.

File: src/applications/claims-status/sass/claims-status.scss
Comment (erikphansen) [inline src/applications/claims-status/sass/claims-status.scss]: This prevents the phone number from wrapping across two lines. This behavior is approved by @emilywaggoner.

**Review by cehsu — APPROVED** [review]
LGTM, pending @emilywaggoner and @peggygannon approval.

File: src/applications/claims-status/sass/claims-status.scss
Comment (cehsu) [reply src/applications/claims-status/sass/claims-status.scss]: It looks like we should set up a ticket to better organize the styles, it seems like edu styles shouldn't be affected by claims styles? Also, does this change affect any claims behavior? If so, we should document that as well via screenshot in this PR for approval.

**Review by peggygannon — COMMENTED** [review]
@erikphansen HI, Erik, I made a couple additional proofread tweaks. Otherwise, the content updates look  good.

File: src/applications/claims-status/sass/claims-status.scss
Comment (erikphansen) [reply src/applications/claims-status/sass/claims-status.scss]: Yes, the Sass organization could use a lot of love, based on my brief time with it. That's a dangerous game to play if there aren't really good visual regression tests in place before starting the process. But it's something that I'd like to do. This change will probably affect other phone numbers on the site. But that is a good thing because a phone number probably shouldn't ever wrap to a new line.

**Comment by erikphansen** [comment]
@peggygannon cool, your changes to that file are all that are needed so we should be good there. Unless you'd like me to update the screenshots to show your latest tweaks?

**Comment by peggygannon** [comment]
@erikphansen Yes, that would be great. I'll give the updated screenshots a quick look before we merge this into staging.

File: src/applications/claims-status/sass/claims-status.scss
Comment (cehsu) [reply src/applications/claims-status/sass/claims-status.scss]: Thanks @erikphansen could you please add a comment to this file that we should move these to a shared style file, and out of this application specific file?

**Comment by erikphansen** [comment]
Updated screenshot showing changes from @peggygannon 

screen shot 2018-07-23 at 4 46 44 pm

**Comment by peggygannon** [comment]
@erikphansen OK, thanks. Looks good.

**Review by peggygannon — APPROVED** [review]

**Review by emilywaggoner — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**
